src/pages/LandingPage/LandingPage.scss

```diff
@@ -1,14 +1,26 @@
+@import "styles/mixins.scss";
+@import "styles/mediaQueries.scss";
+
 .landing-page {
   height: 100vh;
   background-size: cover;
```

Comment:
Should we do this with tailwind?

https://tailwindcss.com/docs/background-size/#app

Reply:
background size, yes, media queries no.

We're going to need the media queries to match tailwind with scss.

Reply:
Yeah, the comment was related to background size.

Reply:
Done